shakespeare.cabal

```diff
@@ -1,5 +1,5 @@
 name:            shakespeare
-version:         2.1.1
+version:         2.1.0
```

Comment:
I would expect this number to go up, not down.

Reply:
I think it can be changed to 2.1.2